src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam is selected');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  function getTitle(item) {
+    return value.startsWith(item) ? setValue('No goods selected') : setValue(`${item} is selected`);
```

Comment:
Function is actually redundant. If the selectedGood is an empty string, then display "No goods selected" message. 
In all other cases display the "${selectedGood} is selected" message